gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
+Establish a marketplace by creating a website(s) that lists InnerSource project needs as "gigs". 
+You can attract potential contributors by posting gigs with explicit requirements (amount of time, skills). People can sign up for a gig and garner approval from their management for those explicit requirements. Individual contributors are then awarded points for the gigs they complete. 
+This makes it explicit what the commitment is and that management agrees with it while encouraging and tracking involvement.
+
+## Problem
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other and connect.
```

Comment:
Based on the context and forces it sounds like there are a few other problems that this pattern helps to solve (e.g. estimating commitment and tracking achievements).

Reply:
Exactly.  The "tracking achievements" part is probably the most compelling.  Although being able to communicate exactly how much time you are expected to spend on the task is a huge selling point for those looking for approval from their direct management to get involved.

Reply:
That's cool.  I (personally) think it would be fine to list out multiple problems that this pattern addresses.

Reply:
Suggestion from the group: split this pattern in two:
1. Publish InnerSource projects on easily accessible website
2. Establish Git Marketplace to ....

from @rshanmer : The _Git Marketplace_ seems to solve multiple problems. Find stuff to work on. Also making mgmt. comfortable with their people working on InnerSource management.
from @stephenmccall : The original focus of the _Git Marketplace_ was the performance management directed toward people.
from @russ: Feels that the patter should on performance management
frpm @daniel: Does this depend on existence of trusted committers, contribution guidelines e.t.c.?
from @stephenmccall : These topics are closely intertwined. 
from @daniel: Also closely linked to problem of findability
from @NewMexicoKid and @dicortazar : let's add links to related patterns whereever the related patterns are mentioned.